highlighting_test.go

```diff
@@ -6,7 +6,9 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/alecthomas/chroma"
 	chromahtml "github.com/alecthomas/chroma/formatters/html"
+	"github.com/google/go-cmp/cmp"
```

Comment:
I think it's enough to just use the standard testing library.

Reply:
Yes, but in such a long block of text(60 lines in this case) this makes it much easier to see what lines are different.

Since the purpose of testing is to be able to easily see what's wrong, I think this is useful.

Reply:
Thoughts?

If you insist, I can remove it, but I think it is useful.

Reply:
What do you say?

Reply:
Using go-cmp is something overkill for the purpose.
We need just a text diff.  We do not need almost functionalities in go-cmp for our purpose.

I'm a minimalist who loves simplicity.
I've added a simple diff function in goldmark/testutil.

https://github.com/yuin/goldmark/commit/16e27ac471a4790d8b1aaed28b37961af55f1230

Could you replace go-cmp with `DiffPretty` ?

Reply:
Done